### DISCUSSION TITLE
Printout member variables

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#17647] [Q&A General] [answered]
Hi MOOSE TEAM,
I am adding member variables following the Example 3 : Multiphysics coupling.
I would like to printout member variables for the confirmation.
For example, I would like to printout "_grad_some_variable" in the Example 3.
Please help me.

ExampleConvection.h in Example 3:
const VariableGradient & _grad_some_variable;

ExampleConvection.C in Example 3 :
params.addRequiredCoupledVar("some_variable", "The gradient of this variable will be used as the velocity vector.");

 _grad_some_variable(coupledGradient("some_variable"))

ex03.i in Example 3:
[Kernels]
[./conv]
    type = ExampleConvection
    variable = convected
    # Couple a variable into the convection kernel using local_name = simulationg_name syntax
    some_variable = diffused
  [../]
[]

Sincerely,
Thomas

**Comment by GiudGiud** [comment, marked as answer]
Hi Thomas

You can look at [VariableGradientCompoent](https://mooseframework.inl.gov/source/auxkernels/VariableGradientComponent.html) to output the gradient to an AuxVariable, then in an exodus file and visualize it there.

Best,
Guillaume
